src/renderer/scss/component/_tooltip.scss

```diff
@@ -105,14 +134,12 @@
   &::after {
     top: 17px;
     right: 100%; // To the left of the tooltip
-
-    border-color: transparent $lbry-gray-5 transparent transparent;
     margin-top: -5px;
   }
 }
 
 .tooltip--top .tooltip__body {
-  bottom: 100%;
+  bottom: 120%;
```

Comment:
Does this always work?

Reply:
Yep, although at this point we only have 1 tooltip with the direction of "top"